qiskit/dagcircuit/dagcircuit.py

```diff
@@ -826,10 +827,15 @@ def _full_pred_succ_maps(self, pred_map, succ_map, input_circuit,
         return full_pred_map, full_succ_map
 
     def __eq__(self, other):
-        if (
-                self.global_phase != other.global_phase
-                or self.calibrations != other.calibrations
-        ):
+        try:
```

Comment:
Can you add a comment explaining the need for the `try`?

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/5682/commits/0d2187498573c5840e1184cf9fe7f2429af13dc6